src/Traits/Util.php

```diff
@@ -42,4 +42,26 @@ public function launchAppiumApp()
 
         return $response;
     }
+
+    /**
+     * DESC
+     * @param $latitude
+     * @param $longitude
+     * @param $altitude
+     * @return mixed
+     * @usage $this->setLocation(100,150,200);
+     * @author Anoop Ambunhi <anoop.nair@tajawal.com>
+     *
+     */
+    public function setLocation($latitude, $longitude, $altitude)
+    {
+        $data = [
+            $location = [
```

Comment:
@anooptajawa convert this to an associative array.

Reply:
@zeeshanu Changed to associative array